src/vuex/getters.js

```diff
@@ -16,6 +16,16 @@ export const stakingDenom = state => {
   )
   return filteredNetwork ? filteredNetwork.stakingDenom : ``
 }
+export const findNetwork = state => value => {
```

Comment:
please read up on getters. this should be a computed value not a function. how it is written rn it should just move to connection.js

Reply:
But how is it possible? We need to select by value.

Reply:
> how it is written rn it should just move to connection.js

Reply:
what I mentioned before was changing the `network` getter to actually be the networks object and not just the network id. This can be a getter based on the network id defined in connection.js